### ISSUE TITLE
ECMScalibration and ECMSCyclicVoltammogram seem to not be compatible

### ISSUE THREAD

**Issue by spoeka22** [ixdat/ixdat#22]
I tried to run a calibration over a selected t_span of an ECMSCyclicVoltammogram object, lets callit ecms_cvmeas:
ecms_cvmeas.ecms_calibration('CO2', 'M44', n_el=+2, tspan=[700, 800], tspan_bg=[600,700])
results in following warning: 
ixdat\src\ixdat\techniques\ec_ms.py:121: RuntimeWarning: invalid value encountered in double_scalars
  F = Y / n
and an output without F: MSCalResult(name=CO2_M44, mol=CO2, mass=M44, F=nan)

if on the other hand I run the calibration before converting the ECMSMeasurement to an ECMSCyclicVoltammogram object, the calibration works fine, if I then use cvmeas=ecms_cvmeas.as_cv() the (calibrated ?) CMSCyclicVoltammogram object (cvmeas) doesnt have any cycles (i.e. if I then call cvmeas[0] (or higher indices), it returns None), but the data itself is there.

I can work around this by just selecting the cv cycles by time instead of cycle, of course, but it's just less convenient. :)

**Comment by spoeka22** [comment]
Also, slightly relayed, I noticed that in the code both voltammagram (with an "a", incorrect spelling) and voltammogram (with an "o", the correct spelling) are being used. I understand that changing all the incorrectly spelled occurrences might be a problem for backwards compatibility, but having both spellings in there in different places is a bit confusing. :)

**Comment by KennethNielsen** [comment]
Thanks for reporting this. Can you share a script (and the needed data files) that reproduces the issue?

**Comment by KennethNielsen** [comment]
Can I persuade you to report the spelling as a separate issue?

> ... I understand that changing all the incorrectly spelled occurrences might be a problem for backwards compatibility, but having both spellings in there in different places is a bit confusing. :)

There are usually ways around that.

**Comment by spoeka22** [comment]
@KennethNielsen I uploaded some code plus data here: https://github.com/spoeka22/info_issue_reports
Hope that works for you.
Note that I've been using the [fuzhou] branch when I first observed this issue, but I also tested on [user_ready] now and it behaves the same.

**Comment by ScottSoren** [comment]
@AnnaWiniwarter , is this okay to close after #50 ?

**Comment by AnnaWiniwarter** [comment]
Yes, this is solved for me after #50 (with new documentation on the importance of using `.redefine_cycle()`)
